### PR TITLE
Clean up the mess

### PR DESCRIPTION
**Pull request by vegasten**

### CODE REVIEW

**Comment by kjosavik** [comment]
LGTM

File: .github/workflows/clean.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: Clean Workspace
+on: [push, pull_request]
```
Comment (kjosavik) [inline .github/workflows/clean.yml:2]: vurder å legge til [issue_comment, gollum, issues, page_build,release]

File: .github/workflows/clean.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+jobs:
+  clean:    
+    runs-on: [ubuntu-latest] 
+    steps:
+      - uses: AutoModality/action-clean@v1.0.1
+  
+
```
Comment (flyrev) [inline .github/workflows/clean.yml]: Do we need *all* these blank lines?

I reckon we only need 3-4.

**Review by flyrev — DISMISSED** [review]

**Review by flyrev — APPROVED** [review]

**Comment by kjosavik** [comment]
Denne må merges

**Review by kjosavik — APPROVED** [review]

**Pull request merged**